test/session_spec.lua

```diff
@@ -168,6 +169,23 @@ test_session("Session using ChidProcessStream", function ()
   return Session.new(proc_stream)
 end)
 
+describe("ChildProcessStream", function()
+  it("can capture exit code", function()
+    local proc_stream = ChildProcessStream.spawn({
+      nvim_prog, '-u', 'NONE', '--embed',
+    })
+
+    local session = Session.new(proc_stream)
+    local ok = session:request "nvim_get_api_info"
+    session:request("nvim_command", "qall!")
+
+    assert.is_true(ok)
```

Comment:
dunno this keeps popping up.

```suggestion
    assert(ok)
```